components/checkbox/checkbox.js

```diff
@@ -8,16 +8,24 @@ function Checkbox(props) {
   const {
     checked,
     children,
+    dataAttrs = {},
     disabled,
     name,
     onChange,
+    ...otherProps
```

Comment:
You are right. Moreover, I think that if we add possibility to pass `otherProps` to all components, we will get more flexibility, and, of course, we won't need ane `dataAttributes` in this case. So, I'll better remove `dataAttributes` from checkbox (because it was me who added them). But we can't remove `dataAttributes` from everywhere until we are sure the nobody uses it. It would be nice if we can make `dataAttributes` deprecated and fully switch to `otherProps` for flexibility.